sapphire/utils.py

```diff
@@ -20,7 +20,7 @@
 ERR = [-1, -999]
 
 #: Speed of light in vacuum in m / ns.
-c = 0.299792458
```

Comment:
I think sticking with `c` is the best solution. Perhaps then eliminating `a`, `b`, `c` variables (or just leave them).

@153957: I would like to release `sapphire-1.3` somewhere this week, so I can get the singles processing on pique. So let's decide on this and merge.